packs/spell-effects/aura-bane.json

```
@@ -0,0 +1,54 @@
+{
+    "_id": "8vKTSlpWnGPnL1UB",
+    "img": "systems/pf2e/icons/spells/bane.webp",
+    "name": "Aura: Bane",
+    "system": {
+        "badge": {
+            "labels": [
+                "10 ft.",
+                "20 ft.",
+                "30 ft.",
+                "40 ft.",
+                "50 ft.",
+                "60 ft."
+            ],
+            "loop": false,
+            "type": "counter",
+            "value": 1
+        },
+        "description": {
+            "value": "<p><span style=\"font-family:Roboto, sans-serif\">Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bane]</span></p>\n<p><span style=\"font-family:Roboto, sans-serif\">You fill the minds of your enemies with doubt. Enemies in the area must succeed at a Will save or take a –1 status penalty to attack rolls as long as they are in the area. Once per round on subsequent turns, you can Sustain the spell to increase the emanation's radius by 10 feet and force enemies in the area that weren't yet affected to attempt another saving throw.</span></p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bane]</p>\n<p>Once per round on subsequent turns, you can Sustain the spell to increase the emanation's radius by 10 feet.</p>"
```

Don't need span cruft from copy/paste and effects should only say what they do.

That said, I'm not big on blank tracking effects like this

---

Good point.

I am hoping this is the first of many steps to getting some proper tracking for this spell.

Still trying to figure out how to approach this from a programmatical standpoint